kolibri/deployment/default/cache.py

```diff
@@ -0,0 +1,48 @@
+import copy
+import sys
+
+from kolibri.utils.conf import OPTIONS
+
+cache_options = OPTIONS["Cache"]
+
+# Default to LocMemCache, as it has the simplest configuration
+default_cache = {
+    "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+    # Default time out of each cache key
+    "TIMEOUT": cache_options["CACHE_TIMEOUT"],
+}
+
+built_files_prefix = "built_files"
+
+built_files_cache = {
+    "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+    # Default time out of each cache key
+    "TIMEOUT": cache_options["CACHE_TIMEOUT"],
+}
+
+
+if cache_options["CACHE_BACKEND"] == "redis":
+    if sys.version_info.major == 3 and sys.version_info.minor < 5:
+        raise RuntimeError(
+            "Attempted to use redis cache backend with Python 3.4, please use Python 2.7 or 3.5+"
+        )
+    base_cache = {
+        "BACKEND": "redis_cache.RedisCache",
+        "LOCATION": cache_options["CACHE_LOCATION"],
+        # Default time out of each cache key
+        "TIMEOUT": cache_options["CACHE_TIMEOUT"],
+        "OPTIONS": {"PASSWORD": cache_options["CACHE_PASSWORD"]},
+    }
+    default_cache = copy.deepcopy(base_cache)
+    default_cache["OPTIONS"]["DB"] = 0
+    built_files_cache = copy.deepcopy(base_cache)
+    built_files_cache["OPTIONS"]["DB"] = 1
```

Comment:
why are we hardcoding the redis databases to be 0 & 1 and not having them as optional?
Thinking of a server where redis is already in use this might taint it. As a sysadmin I would like to be able to use different dbs for different applications.?

Reply:
Good point - my impression from the `redis-cache` docs was that each Django cache needed a separate database number, but would be good to allow a custom starting number to be specified?

Reply:
I think so. I have worked previoulsy in servers where a single redis server was used by different applications and having the option to select the db is important.

Reply:
OK, can add. I just only wanted to add one option, so that people don't have to configure each database number for each cache separately.